tests/transformers/test_transformers_autolog.py

```diff
@@ -282,6 +283,10 @@ def my_hp_space_optuna(trial):
     )
 
 
+@pytest.mark.skipif(
+    Version(transformers.__version__).is_devrelease,
+    reason="fails due to issue: https://github.com/huggingface/setfit/issues/564",
+)
```

Comment:
```suggestion
... = pytest.mark.skipif(
    Version(transformers.__version__).is_devrelease,
    reason="fails due to issue: https://github.com/huggingface/setfit/issues/564",
)
@...
```

you can create a decorator and reuse it.